src/main/java/uk/gov/cshr/civilservant/controller/OrganisationalUnitController.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.cshr.civilservant.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.rest.webmvc.RepositoryRestController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import uk.gov.cshr.civilservant.domain.OrganisationalUnit;
+import uk.gov.cshr.civilservant.service.OrganisationalUnitService;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+@RepositoryRestController
+@RequestMapping("/organisationalUnits")
+public class OrganisationalUnitController {
+
+    private OrganisationalUnitService organisationalUnitService;
+
+    @Autowired
```

Comment:
`@Autowired' isn't necessary here. Spring will automatically try to autowire any controllers, services, components etc